### DISCUSSION TITLE
Methods Across Webex Orgs

### DISCUSSION THREAD

**Discussion by kctrey** [kctrey/wxcadm#2] [Ideas]
One of the initial decisions I was faced with was how to handle admins that have access to more than one Org. Even at the Partner level, most actions would be taken within an Org, but there are obviously some cases that may be better handled across all Orgs. For example, if an admin wanted to find the Org that a user was associated with, across many Orgs, Webex-level methods might be useful.

I'm looking for input from anyone who uses this module to see if that would be a benefit or not.

**Comment by vaijanaths007** [comment]
Hi Trey,
Thank you for your efforts on this project and making it publicly available. I started using your project recently and making progress little by little. I have added few sections using your code which are needed for my current project. I am working at partner level and it would be great to have option to access across multiple organizations. I tried to use one method from your code to get voicemail config for users from customer organization and it didn't work. I will try to find solution for that.

**Comment by kctrey** [comment]
As long as your API Token has access to all the Orgs, you should be able to use the Webex.orgs attribute to access all of the Orgs.

For, example, to get the VM config for Webex Calling users in every Org you have access to:

``` python
webex = wxcadm.Webex('Access Token')

for org in webex.orgs:
    for person in org.wxc_people:
        print(person.get_vm_config())
```

If that isn't working for you, let me know. I just tested it with some Orgs that I am able to access and it seems to work.

**Reply by vaijanaths007** [reply]
Hi Trey,
This code looks for VM settings for all users in all organization and it works fine. What I am looking is for just specific organization. As a partner admin, I should have option to specify org name and get the VM config details from that org only.

**Reply by kctrey** [reply]
You can use the Webex.get_org_by_name() method to return only the Org that you want to work with.

``` python
webex = wxcadm.Webex('Access Token')
customer_org = webex.get_org_by_name('Customer Org Name')

for person in customer_org.wxc_people:
    print(person.get_vm_config())
```

**Reply by vaijanaths007** [reply]
Hi Trey,
I got mine working. Thank you for your help.
